libcidr-shopify.rb

```diff
@@ -0,0 +1,19 @@
+class LibcidrShopify < Formula
+  desc 'Libcidr is a library that provides utility functions to handle IP addresses and net blocks.'
+  homepage 'https://www.over-yonder.net/~fullermd/projects/libcidr'
+  url 'http://bazaar.launchpad.net/~fullermd/libcidr/trunk/tarball/302'
+  sha256 '44d99c491a90d01baf0e074f6b5bac48e7eab93e97835819f631ef2a8bd12935'
+
+  patch do
+    url "https://gist.githubusercontent.com/fmejia97/ad72a1f2b5d4b77a2921f32d861850b8/raw/151aabb91fe183ca3d18521c29f90007aa690592/mac_os_libcidr_support.patch"
```

Comment:
Does it make sense that we store the patch somewhere else? Once Francisco's term is over, we have a build that depends on a non-employee not deleting a gist that we have no control to.

Reply:
@charlescng  That is a very good point. We could store the patch in google cloud storage to make it account agnostic.

Reply:
Probably could just be committed right into this repo. There are others on other accounts too. Eg. Nginx patches on marc’s Account.

Reply:
I like the idea of committing the patch to this repo so it becomes version controlled. I'll add a new commit with the patch.

Reply:
url should probabaly be updated to https://raw.githubusercontent.com/Shopify/homebrew-shopify/3e72d2602b76e28036b1c8dded7f61569ba184a4/patch/mac_os_libcidr_support.patch